### PR TITLE
Add GnoNGPL

### PR DESCRIPTION
**Pull request by jaekwon**
This addresses #49.

### CODE REVIEW

**Review by Pipello — COMMENTED** [review]
Is it my screen (hopefully not my eyes getting worse) or is it using old style double spacing after period? 
Anyway I like we are using GnoPL in the end and not Apache 2... fuck the industry ;)

**Review by salmad3 — APPROVED** [review]

**Comment by stevenoruzi** [comment]
it seems that https://www.gno.land/licenses/ is unavailable. 
error: 404 page not found

**Comment by salmad3** [comment]
> it seems that https://www.gno.land/licenses/ is unavailable. error: 404 page not found

- https://github.com/gnolang/gno/blob/master/gno.land/LICENSE.md
- https://github.com/gnolang/gno/blob/master/LICENSE.md

**Comment by jaekwon** [comment]
Ok. We should make a NewTendermint,LLC managed website and put the licenses in there, on a new domain.
As a reminder, NewTendermint,LLC manages the "Gno" brand, but gno.land has the license and autonomy to use gno.land.
The gno.land domain is owned by NewTendermint,LLC, and managed on behalf of gno.land within reason.
(though others can own *gnoland* etc and operate it entirely in accordance with gno.land, as long as it is distinct from "gno"). 
So I will update all the licenses soon to reflect a new domain.
For now we can all agree that www.gno.land/licenses will display only GnoNGPL and GnoGPL licenses published by NewTendermint,LLC; until all of this is resolved.

**Comment by moul** [comment]
The upcoming deployment of the gno.land portal loop will greatly simplify the process of rendering the license on the gno.land/licenses page or create redirection to NT.org.

Also, I can set up a repository to host a collection of static files for NewTendermint.org as a temporary measure before we develop a full HTML homepage. I just need guidance on whether to create a new GitHub organization for this or if we should temporarily use the existing gnolang one.

**Comment by moul** [comment]
@jaekwon FYI, you’re still the only one able to merge this PR; we made “gray” approvals that are just informative.

To start having other people to merge, we need to:
1. To have branch protection (DONE)
2. Have “Write” permissions on the repo. @atomone-hub/mod currently has “Triage” permission which is not enough.
3. To be in the CODEOWNERS file; which is not the case for this new file, since it’s new :)

In other words, we can let the PR open if you want more feedback, but only you will be able to merge it at the end.

**Comment by moul** [comment]
Yes, that's the issue. Without 2, we cannot merge. However, combining 1 and 3 makes 2 controlled and safe.
